### ISSUE TITLE
Licence tempate

### ISSUE THREAD

**Issue by joker314** [BKFighter/bkfighter.github.io#2]
..has a bunch of legal flaws, like lack of apostrophes and definitions, as well as not specifying whixh legal system "law abiding" is referring to

**Comment by BKFighter** [comment]
Thanks for that. I'll update it with those things in mind.

**Comment by BKFighter** [comment]
If you don't mind taking a look at my revisions:
BKFighter Copyright License Template 1.2

SECTION 1
The owners(s) of this software, as listed in section 2, permit you to run/execute the following code but without any 'modifications', except for those from contributors (as listed in Section 2) or as permitted by owner(s) in Section 3. Re-Distribution is strictly not allowed without owner's permission (Section 3). You may not use any code whose origin lies in this project. (Code used from other programs/scripts may be used as permitted by license of said code.) You may not hold owners or contributors at fault for any damages caused by this project/script, monetary or other, as long as script abides [state country]'a laws.

SECTION 2
Owner(s):
[list owners of code, including company if appropriate]
Contributor(s):
[list any who wrote any code whose origin lies in this script/project]

SECTION 3 (optional)
[list extra permissions as stated by owners)

SECTION 4
Definition(s):
Modifications - Any change of code/scripts in project, including minifying and confuscating.

License created and maintained by BKFighter (2016). License free to use and modify, even commercially, with credit to BKFighter. Please mark all modifications of license to users.

**Comment by BKFighter** [comment]
And I haven't decided on a legal system yet.

**Comment by BKFighter** [comment]
@joker314 ^^^

**Comment by BKFighter** [comment]
@joker314

**Comment by BKFighter** [comment]
@joker314 please?

**Comment by joker314** [comment]
Yes, I couldn't find any legal flaws, but does replacing the stuff in square brackets count as a modification of the licence?

**Comment by BKFighter** [comment]
No.

**Comment by joker314** [comment]
Ok, you might want to mention that if you want
